Credit recognition (#674)
EVERSE-ResearchSoftware/RSQKit: pages/tasks/credit_recognition_research_software.md

---

> title: "Credit and recognition for research software and RSEs"

Author: ```suggestion
title: "Credit and recognition for research software and people who write it"
```

> description: "Guidance on making software contributions visible, citable, and verifiable, and on building the evidence needed for career progression as a Research Software Engineer."

Author: ```suggestion
description: "Guidance on making software contributions visible, citable and verifiable, and building the evidence needed for career progression for people who write research code."
```
